metadata-ingestion/setup.py

```diff
@@ -255,6 +255,10 @@ def get_long_description():
     "requests",
 }
 
+mysql = sql_common | {"pymysql>=1.0.2"}
+
+kafka = kafka_common | kafka_protobuf
```

Comment:
do we need kafka_protobuf for the datahub source?

we should only need kafka_common right?

Reply:
Ah I don't know I just took everything. I'll try with just kafka common